### ISSUE TITLE
panos_mgtconfig is setting wrong domain name

### ISSUE THREAD

**Issue by chewdevs** [PaloAltoNetworks/ansible-pan#197]
I used this configuration to set manegement settings:

```
- name: configure management settings (DNS,NTP)
  panos_mgtconfig:
    ip_address: "{{ inventory_hostname }}"
    username: "{{ ansible_user }}"
    password: "{{ ansible_password }}"
    hostname: "{{ inventory_hostname_short }}"
    domain: "{{ domain }}"
    dns_server_primary: "{{ dns_server_primary }}"
    dns_server_secondary: "{{ dns_server_secondary }}"
    timezone: "{{ timezone }}"
    ntp_server_primary: "{{ ntp_server_primary }}"
    ntp_server_secondary: "{{ ntp_server_secondary }}"
  run_once: true
  notify: commit
  tags: system,config
```

with this var:
`
domain: cloud.620nm.net
`

When I run the playbook, it seems OK:
```
TASK [panos : configure management settings (DNS,NTP)] *************************************************************************************
task path: /home/adminsip/Share/git/ansible-role/panos/tasks/system.yml:1
<cloud-fw-y01n-prd.cloud.620nm.net> ESTABLISH LOCAL CONNECTION FOR USER: adminsip
<cloud-fw-y01n-prd.cloud.620nm.net> EXEC /bin/sh -c 'echo ~adminsip && sleep 0'
<cloud-fw-y01n-prd.cloud.620nm.net> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852 `" && echo ansible-tmp-1542041660.39-87999804315852="` echo /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852 `" ) && sleep 0'
Using module file /home/adminsip/Share/git/PF-FW-Virtuels/ansible/roles/PaloAltoNetworks.paloaltonetworks/library/panos_mgtconfig.py
<cloud-fw-y01n-prd.cloud.620nm.net> PUT /home/adminsip/.ansible/tmp/ansible-local-3726HF8K20/tmpKufWQp TO /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852/AnsiballZ_panos_mgtconfig.py
<cloud-fw-y01n-prd.cloud.620nm.net> EXEC /bin/sh -c 'chmod u+x /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852/ /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852/AnsiballZ_panos_mgtconfig.py && sleep 0'
<cloud-fw-y01n-prd.cloud.620nm.net> EXEC /bin/sh -c '/home/adminsip/virtenv/ansible2.7/bin/python /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852/AnsiballZ_panos_mgtconfig.py && sleep 0'
<cloud-fw-y01n-prd.cloud.620nm.net> EXEC /bin/sh -c 'rm -f -r /home/adminsip/.ansible/tmp/ansible-tmp-1542041660.39-87999804315852/ > /dev/null 2>&1 && sleep 0'
NOTIFIED HANDLER panos : commit for cloud-fw-y01n-prd.cloud.620nm.net
changed: [cloud-fw-y01n-prd.cloud.620nm.net] => {
    "changed": true, 
    "invocation": {
        "module_args": {
            "api_key": null, 
            "commit": true, 
            "devicegroup": null, 
            "dns_server_primary": "10.27.53.3", 
            "dns_server_secondary": "10.94.53.3", 
            "domain": "cloud.620nm.net", 
            "hostname": "cloud-fw-y01n-prd", 
            "ip_address": "cloud-fw-y01n-prd.cloud.620nm.net", 
            "login_banner": null, 
            "ntp_server_primary": "10.27.53.3", 
            "ntp_server_secondary": "10.94.53.3", 
            "panorama_primary": null, 
            "panorama_secondary": null, 
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", 
            "timezone": "Europe/Paris", 
            "update_server": null, 
            "username": "iac_user"
        }
    }, 
    "msg": "okey dokey"
}
```

But in the firewall, the domain is configured with the hostname and not the value of the `domain` variable.

**Comment by freakinhippie** [comment]
@chewdevs Good catch, I've fixed the bug. I'm submitting a PR for it now.

**Comment by shinmog** [comment]
This is fixed by #198
